ui/helpers/utils/nfts.ts

```diff
@@ -28,3 +37,18 @@ export const getNftImageAlt = ({ name, tokenId, description }) => {
   const altText = description ?? `${name ?? ''} ${tokenId ?? ''}`.trim();
   return nftTruncateAltText(altText, NFT_ALT_TEXT_MAX_LENGTH);
 };
+
+type NFTImage = string | string[] | undefined | null;
+export const getNftImage = (image: NFTImage): string | undefined => {
+  if (typeof image === 'string') {
+    return image;
+  }
+
+  // TEMP - UI/UX change to support multiple NFT images
+  // or we would need to highly specific logic to better determine which NFT image to view
```

Comment:
Remove this comment, we have decided not to pursue this, and we will always default to the first image.